lib/oneview-sdk/resource/api300/c7000/logical_switch.rb

```diff
@@ -45,6 +49,15 @@ def self.get_internal_link_set(client, name)
           results = get_internal_link_sets(client)
           results.find { |internal_link_set| internal_link_set['name'] == name }
         end
+
+        # Updates scopes uris of a given logical switch resource
+        # @param [String] operation The type of operation: one of "add", "remove" or "replace".
+        # @param [String] path The path operation for use. The exact meaning depends on the type of operation.
+        # @param [String] value The value to add or replace for "add" and "replace" operations. Not used by "remove".
+        # @note This attribute is subject to incompatible changes in future release versions, including redefinition or removal.
+        def patch(operation, path, value = nil)
```

Comment:
I don't know if this updated comment would be helpful. I really think the user will have to look at the REST API to know how and what to put in this operation. Maybe we could remove this method from here, and leave it with the standard patch comment. :thought_balloon: